packages/react-storage/src/components/StorageBrowser/composables/DataTable/dataCells/DateDataCell.tsx

```diff
@@ -6,18 +6,19 @@ import { STORAGE_BROWSER_BLOCK_TO_BE_UPDATED } from '../../../constants';
 export interface DateDataCellProps {
   content: {
     date?: Date;
+    text?: string;
   };
 }
 
 export const DateDataCell = ({
   content,
 }: DateDataCellProps): React.JSX.Element => {
-  const { date } = content;
+  const { text } = content;
   return (
     <ViewElement
       className={`${STORAGE_BROWSER_BLOCK_TO_BE_UPDATED}__table-date-data-cell`}
     >
-      {date?.toLocaleString()}
+      {text}
```

Comment:
Since text is optional, would the toLocaleString be a reasonable fallback value?

Reply:
Yeah but at that point there is nothing more we can do for this cell. 🪦